cvm-python-app-remoteattest/attestationclientdeploy.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: attestation
+  labels:
+    app: attestation
+spec:
+  containers:
+  - image: lizzhaacr.azurecr.io/attestation-client:latest
```

Comment:
this image is in private ACR, you can use the public image instead
`mcr.microsoft.com/acc/samples/cvm-attestation:1.0`